src/registrar/templates/application_dotgov_domain.html

```diff
@@ -2,24 +2,22 @@
 {% load static field_helpers url_helpers %}
 
 {% block form_instructions %}
-  <p>Before requesting a .gov domain, <a class="usa-link" rel="noopener noreferrer" target="_blank" href="{% public_site_url 'domains/choosing' %}">please make sure it
-  meets our naming requirements</a>. Your domain name must:
+  <p>Before requesting a .gov domain, please make sure it meets <a class="usa-link" rel="noopener noreferrer" target="_blank" href="{% public_site_url 'domains/choosing' %}">our naming requirements</a>. Your domain name must:
     <ul class="usa-list">
       <li>Be available </li>
-      <li>Be unique </li>
       <li>Relate to your organization’s name, location, and/or services </li>
       <li>Be clear to the general public. Your domain name must not be easily confused
       with other organizations.</li>
     </ul>
   </p>
 
+  <p>Names that uniquely apply to your organization are likely to be approved over names that could also apply to other organizations. In most instances, this requires including your state’s two-letter abbreviation.</p> 
```

Comment:
These paragraphs are missing the italicized sections

Reply:
```suggestion
  <p>Names that _uniquely apply to your organization_ are likely to be approved over names that could also apply to other organizations. In most instances, this requires including your state’s two-letter abbreviation.</p> 
```